content/facets.md

```diff
@@ -0,0 +1,138 @@
+# 12 Factor Facets
+
+While the 12 factors serve as helpful specific guidelines for development teams, the rationale behind the factors is to enable a set of application-platform interactions which simplify the operation of applications which follow the recommendations in the factors.  These interactions can be broadly grouped by the platform behaviors they enable, which serve as “facets” of operation where the 12 factors simplify maintenance.  The facets enable applications which are:
+
+* **Continuously Deployable**: it is easy to deploy new software versions in existing environments
+* **Composable**: it is easy to set up existing software versions in new environments
+* **Scalable**: it is easy to manage the software within an environment
+* **Observable**: it is easy to debug the software when running in an production environment
+
+The following sections will map how specific factors contribute to the different facets of application operations.  Applications which apply the contributing factors for a particular facet may be able to derive some benefits from the 12 factors even if they cannot satisfy all of the factors.
```

Comment:
May want to call out that 12-factor is stateless, and the overall system can make a `Consistency | Availability` choice based on the backing services.  Since C|A is a state-management question, building in a 12-factor manner makes the C|A choice a no-op at the app architecture level, and instead inherits the decisions from state-handling backing services.